applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -243,22 +243,16 @@ static const char *const cmd_type_str[] = {
 BUILD_ASSERT(ARRAY_SIZE(cmd_type_str) == CLOUD_CMD__TOTAL);
 
 static struct cloud_sensor_chan_cfg sensor_cfg[] = {
```

Comment:
not all of the `CLOUD_CHANNEL_`s have a cfg value, so it would be adding unused array elements.
if you'd prefer that to the for loops, i can change it.

Reply:
Oh okay, let's just keep the loops